.github/workflows/simorgh-misc-checks.yml

```diff
@@ -31,6 +31,12 @@ jobs:
       - name: Chromatic UI Tests
         run: npx chromatic test run --build-script-name build:storybook --exit-once-uploaded --no-interactive --project-token "$CHROMATIC_APP_CODE"
 
+      - name: Code Climate Test Coverage
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter	
+          chmod +x ./cc-test-reporter
+          ./cc-test-reporter after-build -t lcov --debug --exit-code 0
+
```

Comment:
Does this check maybe belong with the unit tests - I noticed in the old script, it ran `npm run test:local` - we would rather avoid doing this again unless absolutely necessary

Reply:
good spot, giving that a go now.